### DISCUSSION TITLE
[Feature Request]: Allow hiding Dumps and Mail windows with cmd+h on macOS

### DISCUSSION THREAD

**Discussion by marcusmoore** [beyondcode/herd-community#487] [Ideas]
### Operating system version

macOS Sonoma 14.3.1

### System architecture

ARM64 (M1, M2, etc)

### Herd Version

1.5.0 (Build: 23)

### Bug description

Hi there. I have a non-breaking issue that might be better suited as a feature request but I'm calling it a bug since Herd strays from the "norm" for macOS window management.

Herd windows for Dumps and Mail don't work with the traditional macOS `cmd+h` keyboard shortcut to hide the application window (this isn't an issue with Logs and Services windows).

What I mean is that macOS applications usually allow you to "hide" the application windows by pressing `cmd+h`. It's an option that appears in the app's menu:
![Brave menu](https://github.com/beyondcode/herd-community/assets/1141514/343e9a8b-754a-4fc4-a09e-4feb12e2df3b)

But that option is not available for Herd's Dumps and Mail windows:
![Herd's menu](https://github.com/beyondcode/herd-community/assets/1141514/324940c8-16ae-43fe-801e-1d2ad167bc77)

It would be great if that was implemented because I often use `cmd+h` to hide an app so I can peek at the application it is covering.

Thanks and keep up the awesome work!

**Comment by sschlein** [comment]
That definitely makes sense and we'll have a look as soon as Herd for Windows is released and we can squeeze in some time for smaller improvements.

**Reply by dhaaker** [reply]
Any update on whether this might get implemented? Would be super useful.

Thanks for all the great work!
